README.org

```diff
@@ -116,5 +108,10 @@ Copyright © 2013 - 2016 to the anubad Authors.
   If you wish to contribute this project, there is much thing you can
   do beyond programming, to get started read [[https://github.com/foss-np/anubad/tree/experimental/CONTRIBUTING.org][this]].
 
-  Any code Contributions to anubad should be made via of GitHub pull
+  Any code Contributions to anubad should be made via GitHub pull
   requests.
+
+[[http://i.creativecommons.org/l/by/3.0/88x31.png]]
+
+Copyright © 2013 - 2016 to the anubad Authors.
```

Comment:
@samundra same here 80 char is the norm,

It might be because its not the `markdown` but `org-mode`.

**emacs** does the visual thingi and maintain 80 char limit automatically,

links are usually sorted like in _rendered form_ that might be the reason you got confused.

it kinda looks like this..

![screenshot132](https://cloud.githubusercontent.com/assets/1684754/16574896/6f8befe2-42a3-11e6-8740-dcd677de74b2.png)